### PR TITLE
howto_release.md: updated for G8

### PR DESCRIPTION
**Pull request by neteler**
- various updates from G7.x to G8

### CODE REVIEW

File: doc/howto_release.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Now check if configure still works.
 If yes, submit to git:
 
 ```bash
-git checkout -b config_sub_update_r78
+git checkout -b config_sub_update_r80
 git add config.guess config.sub configure
 git commit -m"config.guess + config.sub: updated from http://git.savannah.gnu.org/cgit/config.git/plain/" config.guess config.sub configure
```
Comment (veroandreo) [inline doc/howto_release.md:64]: Not related to changes in this PR, but why are the files added again here in the `git commit`?

File: doc/howto_release.md [hunk 2]
```diff
@@ -271,15 +272,15 @@ To easily generate the entries for the trac Wiki release page, use the `git log`
...
 # prepare for trac Wiki release page (incl. PR trac macro)
-git log --oneline --after="2020-10-05" | cut -d' ' -f2- | sed 's+^+ * G78:+g' | sed 's+(#+(PR:+g' | sort -u
+git log --oneline --after="2021-10-10" | cut -d' ' -f2- | sed 's+^+ * G80:+g' | sed 's+(#+(PR:+g' | sort -u
 ```
 
 - store changelog entries in trac, by section:
-    - <https://trac.osgeo.org/grass/wiki/Release/7.8.x-News>
-    - <https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78>  <- add content of major changes only
+    - <https://trac.osgeo.org/grass/wiki/Release/8.0.x-News>
+    - <https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80>  <- add content of major changes only
```
Comment (veroandreo) [inline doc/howto_release.md:284]: Shall we keep these two if the approach generating the "nice" changelog in github works out?

File: doc/howto_release.md [hunk 3]
```diff
@@ -301,42 +302,29 @@ Software pages:
...
+     set MAJOR=8
+     set MINOR=0
      set PATCH=0RC1
 ```
 
 - Update addons (grass_addons.sh) rules, eg.
 
 ```
-     compile $GIT_PATH/grass7 $GISBASE_PATH/grass780RC1   $ADDON_PATH/grass780RC1/addons
+     compile $GIT_PATH/grass8 $GISBASE_PATH/grass800RC1  $ADDON_PATH/grass800RC1/addons
```
Comment (veroandreo) [inline doc/howto_release.md:321]: good to have this in place but the `grass8` branch in the addons repo has not been created yet

**Review by veroandreo — APPROVED** [review]
Seems good to me, some comments here and there. 

I'd like to revise/reduce the list of websites to post the release announcements. Most of them do not work anyway, at least that was my experience. I'd personally only keep our mailing lists, social media and OSGeo news.

File: doc/howto_release.md (hunk 1 above)
Comment (neteler) [reply doc/howto_release.md]: Right, fixed in 3f994c8eec5ac42eaa8f7170ccb40cf358e7624c

File: doc/howto_release.md (hunk 2 above)
Comment (neteler) [reply doc/howto_release.md:284]: Once it really works, we may change it. This is a "living" document...

File: doc/howto_release.md (hunk 3 above)
Comment (neteler) [reply doc/howto_release.md:321]: True but @wenzeslaus planned to do that these days (see https://lists.osgeo.org/pipermail/grass-dev/2021-November/095449.html). I'd prefer to get rid of G7.x in this document.

**Comment by nilason** [comment]
Taking the opportunity to ask a quick question on something just tangential to this PR. I noticed the version in `include/VERSION` is the same on main branch and on the releasebranch_8_0. Is that intentional?

File: doc/howto_release.md (hunk 3 above)
Comment (wenzeslaus) [reply doc/howto_release.md:321]: Created, so yes, it's the time for updates.

**Comment by neteler** [comment]
@nilason well, out of laziness so far, since `main` will never be released directly...

**Pull request merged**